web/index.html

```diff
@@ -63,5 +60,15 @@ <h1>Title of the chart</h1>
       </div>
     </div>
   </footer>
+
+  <!-- Login modal -->
+  <div class="modal-wrapper" id="login-modal">
+    <div class="modal-content">
+      <h5>This demo is meant to be private to Africa's voices.</h5>
+      <p>Please reach out if you like access.</p>
+      <button class="btn btn-danger" id="login-button">Login via Google SSO</button>
```

Comment:
'Login with Google' - is the way that OAuth SSO is usually described to users

Reply:
All the examples in official google docs say "Sign in with Google" - will change it to the recommended one.

Reply:
Fixed